lib/plug/debugger.ex

```diff
@@ -122,7 +122,23 @@ defmodule Plug.Debugger do
 
       def call(conn, opts) do
         try do
-          super(conn, opts)
+          case conn do
+            %Plug.Conn{path_info: ["__debug__", "action"], params: params} ->
```

Comment:
Suggestion, let's have it call "__plug__/__debugger__", so it is unlikely to ever conflict.